data/json/mutations/mutations.json

```
@@ -4613,10 +4613,14 @@
     "id": "WEB_RAPPEL",
     "name": { "str": "Web Diver" },
     "points": 2,
-    "description": "Your webbing is easily strong enough to support your weight.  You'll use it to descend any sheer drops you may encounter.",
+    "description": "Your webbing is easily strong enough to support your weight.  Activate to rappel down any drop you encounter, at the cost of nutrition.",
```

Begone superfluous intensifier!
```suggestion
    "description": "Your webbing is strong enough to support your weight.  Activate to rappel down any drop you encounter, at the cost of nutrition.",
```